agent/ecscni/types_linux.go

```diff
@@ -29,6 +29,11 @@ const (
 	// defaultAppMeshIfName is the default name of app mesh to setup iptable rules
 	// for app mesh container. IfName is mandatory field to invoke CNI plugin.
 	defaultAppMeshIfName = "aws-appmesh"
+	// defaultServiceConnectIfName is the default ifname used for invoking SC CNI plugin.
+	// IfName is a mandatory field to invoke any CNI plugin. For tasks in awsvpc mode, the network configuration
+	// does not need the ifname so this is simply a placeholder. For tasks in bridge mode, the configuration does
+	// require the ifname, and we use "eth0" because that's the default interface for docker bridge container network.
+	defaultServiceConnectIfName = "eth0"
```

Comment:
I don't think this is accurate. There should be no interface names hardcoded in CNI plugins or agent. We have removed the use of ifName from ServiceConnect CNI plugin.

Reply:
That is right. We dont use the ifName within the plugin.

Reply:
Okay will update.